ttools/callbacks.py

```diff
@@ -506,6 +515,22 @@ def batch_end(self, batch_data, train_step_data):
         self.checkpointer.save(filename, extras={"epoch": self.epoch})
```

Comment:
here and elsewhere were `save` is called, we might want to always save the best_val_value for easy loading and treat the "None" case. It could be nice, but not required, to add a "load_best" method to the Checkpointer class.